dev/Dockerfile

```diff
@@ -0,0 +1,22 @@
+from openms/executables:latest
```

Comment:
Similar concerns as @Zethson already pointed out. Also, we tend to use the nf-core/base container for everything that is not on bioconda - people _could_ (not saying you do as we know each other) ship malicious stuff with their own base containers. No way to use the nfcore/base one? 

Not sure about others opinions on this one @nf-core/core  ?